receiver/postgresqlreceiver/scraper_test.go

```diff
@@ -286,6 +301,20 @@ func (m *mockClient) initMocks(database string, databases []string, index int) {
 		}, nil)
 		m.On("getMaxConnections", mock.Anything).Return(int64(100), nil)
 		m.On("getLatestWalAgeSeconds", mock.Anything).Return(int64(3600), nil)
+		m.On("getDatabaseLocks", mock.Anything).Return([]databaseLocks{
+			{
+				relation: "pg_locks",
+				mode:     "AccessShareLock",
+				lockType: "relation",
+				locks:    3600,
+			},
+			{
+				relation: "pg_class",
+				mode:     "AccessShareLock",
+				lockType: "relation",
+				locks:    5600,
+			},
+		}, nil)
```

Comment:
any chance you can get a test going where this returns an error?

Reply:
added